gdx-fireapp-ios/src/pl/mk5/gdx/fireapp/ios/analytics/Analytics.java

```diff
@@ -64,12 +64,7 @@ public void logEvent(String name, Map<String, String> params) {
      */
     @Override
     public void setScreen(final String name, final Class<?> screenClass) {
-        NSOperationQueue.getMainQueue().addOperation(new Runnable() {
-            @Override
-            public void run() {
-                FIRAnalytics.setScreenName(name, screenClass.getSimpleName());
```

Comment:
Should we remove this API altogether from the Core Analytics file ?

Reply:
it's still working for android, and others I guess 🤔